src/python/director/drakevisualizer.py

```diff
@@ -434,6 +439,44 @@ def onPlanarLidar(self, msg, channel):
             polyData.SetVerts(verts)
 
 
+    def onPointCloud(self, msg, channel):
+        pointcloudName = channel.replace('DRAKE_POINTCLOUD_', '', 1)
+
+        polyData = vnp.numpyToPolyData(np.asarray(msg.points), createVertexCells=True)
+
+        # If the user provided color channels, then use them to colorize
+        # the pointcloud.
+        channels = {msg.channel_names[i]: msg.channels[i] for i in range(msg.n_channels)}
+        if "r" in channels and "g" in channels and "b" in channels:
+            colorized = True
+            rgb = vtk.vtkUnsignedCharArray()
+            rgb.SetName("rgb")
+            rgb.SetNumberOfComponents(3)
+            rgb.SetNumberOfTuples(msg.n_points)
+            polyData.GetPointData().AddArray(rgb)
+            for (color_index, color) in enumerate(["r", "g", "b"]):
+                channel = channels[color]
+                for i in range(msg.n_points):
+                    rgb.SetComponent(i, color_index, 255 * channel[i])
```

Comment:
@patmarion is there a better way to do this? Looping over all of the points in Python like this is probably unnecessarily slow (although I haven't had any problems in practice yet)